webapp/app/components/pages/ActionButtons/ActionButtons.jsx

```diff
@@ -0,0 +1,80 @@
+{/*<Button color="danger" onClick={this.toggle}>{this.props.buttonLabel}</Button>*/}
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'reactstrap';
+import MsModal from '../../common/modal/MsModal';
+import UsersForm from '../UsersForm/UsersForm';
+
+class ActionButtons extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      modal: false,
+      modalTitle: '',
+      modalBody: ''
+    };
+
+    this.toggle = this.toggle.bind(this);
+    this.toggleAddModal = this.toggleAddModal.bind(this);
+    this.toggleEditModal = this.toggleEditModal.bind(this);
+    this.toggleDeleteModal = this.toggleDeleteModal.bind(this);
+  }
+
+  toggle() {
+    this.setState({
+      modal: !this.state.modal
+    });
+  }
+
+  toggleAddModal() {
+    this.setState({
+      modalBody: (<UsersForm/>),
+      modalTitle: 'Add User',
+      modalYesLabel: 'Save'
+    }, this.toggle);
+  }
+
+  toggleEditModal() {
+    this.setState({
+      modalBody: (<UsersForm/>),
+      modalTitle: 'Edit User',
+      modalYesLabel: 'Save'
+    }, this.toggle);
+  }
+
+  toggleDeleteModal() {
+    this.setState({
+      modalBody: `Are you sure to delete User ${this.props.user.name}`,
+      modalTitle: 'Delete User',
+      modalYesLabel: 'Delete'
+    }, this.toggle);
+  }
+
+  render() {
+    return (
+      <div>
+        <Button color="primary" onClick={this.toggleAddModal}>Add</Button>{' '}
+        <Button color="info" onClick={this.toggleEditModal} disabled={ this.props.editButtonDisabled }>Edit</Button>{' '}
```

Comment:
I would avoid computed whitespace as a general rule, preffer to style and add desired spacing via Stylesheet, or inline styles either adding padding to the buttons or the desired margin

Reply:
idem